rcgen/CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 
 # Changes
 
+## Release 0.12.1 - January 25th, 2024
```

Comment:
Probably we want to cherry-pick this commit into `main` to keep that `CHANGELOG.md` in-sync once 0.12.1 is published.

Reply:
https://github.com/rustls/rcgen/pull/220